src/pages/project/dpad-logic.js

```diff
@@ -40,5 +40,18 @@ module.exports = {
         showRight: verifyCoordsExist(this.state.pages, getTargetCoords(this.state.zoomedPageCoords, 'right'))
       });
     }
+
+    if (this.state.params.mode === 'play' && this.state.isPageZoomed) {
+      // Pass bounding box of the currently focused page to the DPad for display logic
+      var elFocusedElementGroup = document.querySelector('.page-container:not(.blurred) .element-group');
+
+      if (elFocusedElementGroup) {
+        // There is a potential race condition here:
+        setTimeout(function () {
```

Comment:
So for example here https://github.com/mozilla/webmaker-core/blob/ecbb76ac0245c8ec4889868c6764e1890ebc24db/src/pages/project/cartzoom.js#L48

I'm calculating the max page size based on window and cartesian.width and caresian.height (that's the width and height of one unit, i.e. one page)